### PR TITLE
PIP 15 cards

### PR DESCRIPTION
**Pull request by Simisays**

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/animal_friend.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+Name:Animal Friend
+ManaCost:1 G
+Types:Enchantment Aura
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ Pump
+S:Mode$ Continuous | Affected$ Creature.EnchantedBy | AddTrigger$ AttackTrig | Description$ Enchanted creature has "Whenever this creature attacks, create a 1/1 green Squirrel creature token. Put a +1/+1 counter on that token for each Aura and Equipment attached to this creature other than CARDNAME."
+SVar:AttackTrig:Mode$ Attacks | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ Whenever this creature attacks, create a 1/1 green Squirrel creature token. Put a +1/+1 counter on that token for each Aura and Equipment attached to this creature other than Animal Friend.
+SVar:TrigToken:DB$ Token | TokenScript$ g_1_1_squirrel | RememberTokens$ True | SubAbility$ DBPutCounter
+SVar:DBPutCounter:DB$ PutCounter | Defined$ Remembered | CounterType$ P1P1 | CounterNum$ X | SubAbility$ DBCleanup
+SVar:DBCleanup:DB$ Cleanup | ClearRemembered$ True
+SVar:X:Count$Valid Equipment.Attached,Aura.Attached+!EffectSource
```
Comment (Simisays) [inline forge-gui/res/cardsfolder/upcoming/animal_friend.txt:10]: !EffectSource isn't the correct param to implement "attached to this creature other than CARDNAME"

Thoughts ?

File: forge-gui/res/cardsfolder/upcoming/bighorner_rancher.txt [hunk 2]
```diff
@@ -0,0 +1,11 @@
+Name:Bighorner Rancher
+ManaCost:4 G
+Types:Creature Human Ranger
+PT:2/5
+K:Vigilance
+A:AB$ Mana | Cost$ T | Produced$ G | Amount$ X | SpellDescription$ {T}: Add an amount of {G} equal to the greatest power among creatures you control.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bighorner_rancher.txt:6]: fix description

File: forge-gui/res/cardsfolder/upcoming/bloatfly_swarm.txt [hunk 3]
```diff
@@ -0,0 +1,13 @@
+Name:Bloatfly Swarm
+ManaCost:3 B
+Types:Creature Insect Mutant
+PT:0/0
+K:Flying
+K:etbCounter:P1P1:5
+R:Event$ DamageDone | ActiveZones$ Battlefield | ValidTarget$ Card.Self+counters_GE1_P1P1 | ReplaceWith$ Counters | PreventionEffect$ True | AlwaysReplace$ True | Description$ If damage would be dealt to CARDNAME while it has a +1/+1 counter on it, prevent that damage, remove that many +1/+1 counters from then give each player a rad counter for each +1/+1 counter removed this way.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bloatfly_swarm.txt:7]: fix description

File: forge-gui/res/cardsfolder/upcoming/bottle_cap_blast.txt [hunk 4]
```diff
@@ -0,0 +1,8 @@
+Name:Bottle-Cap Blast
+ManaCost:4 R
+Types:Instant
+K:Improvice
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bottle_cap_blast.txt:4]: typo

File: forge-gui/res/cardsfolder/upcoming/bottle_cap_blast.txt [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+A:SP$ DealDamage | ValidTgts$ Any | NumDmg$ 5 | ExcessSVar$ Excess | SubAbility$ DBToken | SpellDescription$ CARDNAME deals 5 damage to any target.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bottle_cap_blast.txt:5]: need valid excess check

File: forge-gui/res/cardsfolder/upcoming/bottle_cap_blast.txt [hunk 6]
```diff
@@ -0,0 +1,8 @@
+Name:Bottle-Cap Blast
+ManaCost:4 R
+Types:Instant
+K:Improvice
+A:SP$ DealDamage | ValidTgts$ Any | NumDmg$ 5 | ExcessSVar$ Excess | SubAbility$ DBToken | SpellDescription$ CARDNAME deals 5 damage to any target. 
+SVar:DBToken:DB$ Token | TokenScript$ c_a_treasure_sac | TokenAmount$ Excess | SpellDescription$ If excess damage was dealt to a permanent this way, create that many tapped Treasure tokens. (They're artifacts with "{T}, Sacrifice this artifact: Add one mana of any color.")
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/bottle_cap_blast.txt:6]: tapped

File: forge-gui/res/cardsfolder/upcoming/brotherhood_scribe.txt [hunk 7]
```diff
@@ -0,0 +1,9 @@
+Name:Brotherhood Scribe
+ManaCost:1 W
+Types:Creature Human Artificer
+PT:1/3
+A:AB$ PutCounter | Cost$ T | Activation$ Metalcraft | PrecostDesc$ Metalcraft — | CounterType$ ENERGY | CounterNum$ 1 | Defined$ You | SpellDescription$ You get {E} (an energy counter).
+T:Mode$ CounterAddedOnce | ValidPlayer$ You | PlayerTurn$ True | TriggerZones$ Battlefield | CounterType$ ENERGY | Execute$ TrigPumpAll | TriggerDescription$ Whenever you get one or more {E} during your turn, creatures you control get +1/+1 until end of turn.
+SVar:TrigPumpAll:DB$ PumpAll | ValidCards$ Creature | NumAtt$ 1 | NumDef$ 1
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/brotherhood_scribe.txt:7]: wrong valid

File: forge-gui/res/cardsfolder/upcoming/animal_friend.txt (hunk 1 above)
Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/animal_friend.txt:10]: Try `NotDefinedOriginalHost`

Comment (Simisays) [reply forge-gui/res/cardsfolder/upcoming/animal_friend.txt:10]: Thanks working

File: forge-gui/res/cardsfolder/upcoming/almost_perfect.txt [hunk 8]
```diff
@@ -0,0 +1,6 @@
+Name:Almost Perfect
+ManaCost:4 G W
+Types:Enchantment Aura
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ Pump
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/almost_perfect.txt:4]: missing KW

File: forge-gui/res/cardsfolder/upcoming/acquired_mutation.txt [hunk 9]
```diff
@@ -0,0 +1,8 @@
+Name:Acquired Mutation
+ManaCost:2 R
+Types:Enchantment Aura
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ Pump
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/acquired_mutation.txt:4]: missing KW again

File: forge-gui/res/cardsfolder/upcoming/animal_friend.txt [hunk 10]
```diff
@@ -0,0 +1,13 @@
+Name:Animal Friend
+ManaCost:1 G
+Types:Enchantment Aura
+A:SP$ Attach | ValidTgts$ Creature | AILogic$ Pump
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/animal_friend.txt:4]: again

**Review by tool4ever — APPROVED** [review]

**Pull request merged**